test/generator.js

```diff
@@ -77,6 +77,16 @@ describe('generate-password', function() {
 				assert.equal(passwords.length, amountToGenerate);
 			});
 
+			it('should generate strict random sequence that has strictly at least one special symbol from the new pool', function() {
```

Comment:
The test description needs to be updated to something like "sequence avoids excluded characters".